DigitalExcellence/dex-frontend
# Feature: accordion functionality & bugs fixed

## Description

Implemented sorting, and highlight filtering for project overview.
Fixed bug where the user endpoint would get called without the user being authenticated.
Fixed bug where empty search on project overview would cause 400 error

## Type of change


## Checklist


## Steps to Test or Reproduce

Implemented sorting, and highlight filtering for project overview.
1. Go to project overview
2. Use the filter options and see the result

Fixed bug where the user endpoint would get called without the user being authenticated.
1. Start application, unauthorized message does no longer appear

Fixed bug where empty search on project overview would cause 400 error
1. Go to project overview
2. Enter spaces in search box
3. 400 error does no longer appear

## Link to issue

Closes: #171 
Closes: #189 
Closes: #179
